docs/transforms.md

```diff
@@ -25,20 +25,68 @@ A transform consists of 4 parts: type, name, matcher, and transformer. Transform
 ## Transform Types
 There are 3 types of transforms: attribute, name, and value.
 
-**Attribute:** An attribute transform adds to the attributes object on a property. This is for including any meta-data about a property such as it's CTI or other information.
+**Attribute:** An attribute transform adds to the attributes object on a design token. This is for including any meta-data about a design token such as it's CTI or other information.
 
-**Name:** A name transform transform the name of a property. You should really only be apply one name transformer because they will override each other if you use more than one.
+**Name:** A name transform transform the name of a design token. You should really only be apply one name transformer because they will override each other if you use more than one.
 
-**Value:** The value transform is the most important as this is the one that changes the representation of the value. Colors can be turned into hex values, rgb, hsl, hsv, etc. Value transforms have a matcher function that filter which properties that transform runs on. This allows us to only run a color transform on only the colors and not every property.
+**Value:** The value transform is the most important as this is the one that changes the representation of the value. Colors can be turned into hex values, rgb, hsl, hsv, etc. Value transforms have a matcher function that filter which properties that transform runs on. This allows us to only run a color transform on only the colors and not every design token.
```

Comment:
> Value transforms have a matcher function that filter which properties that transform runs on.

Value transforms have a matcher function that filter which tokens that transform runs on.